### PR TITLE
fix(gatsby-nodejs): duplicate Dot in File Ext

### PR DESCRIPTION
**Pull request by 239yash**
## Description
Removed extra dot[.] in the extensions.

## Related Issues
#455

### CODE REVIEW

File: packages/gatsby-source-airtable/gatsby-node.js [hunk 1]
```diff
@@ -297,7 +297,7 @@ const localFileCheck = async (key, row, { createNodeId, createNode, store, cache
         let extension = airtableFile.ext;
         if (!extension) {
           const deriveExtension = mime.getExtension(attachment.type); // unknown type returns null
-          extension = deriveExtension ? `.${deriveExtension}` : undefined;
```
Comment (moonmeister) [inline packages/gatsby-source-airtable/gatsby-node.js:300]: I think you need to leave this in once.

**Comment by 239yash** [comment]
@moonmeister 
Please check the changes now

File: packages/gatsby-source-airtable/gatsby-node.js [hunk 2]
```diff
@@ -306,7 +306,7 @@ const localFileCheck = async (key, row, { createNodeId, createNode, store, cache
           cache,
           createNode,
           createNodeId,
-          ext: extension ? `.${extension}` : undefined,
```
Comment (moonmeister) [inline packages/gatsby-source-airtable/gatsby-node.js:309]: That's a good step. But from the issue, you indicated duplicate dots in the filename before the extension(e.g. `example..ext`). If I understand this code correctly that's happening when the `airtableFile.ext` doesn't exist the if statement tries to derive the file extension. In that derivation, it adds the `.ext` prefix. But we added a second one on line 309. Thus causing the bug. 

You might be correct but you need to run this code with a test site to confirm that we're getting `example.ext` and not `exampleext` as I suspect your proposed change is doing. 

I believe the template literal needs to remain on line 309.

**Review by moonmeister — CHANGES_REQUESTED** [review]

File: packages/gatsby-source-airtable/gatsby-node.js (hunk 2 above)
Comment (moonmeister) [reply packages/gatsby-source-airtable/gatsby-node.js:309]: Sorry for not being more clear the first time.

Comment (239yash) [reply packages/gatsby-source-airtable/gatsby-node.js:309]: Hi @moonmeister, Sorry for the delay. Can you please help to test this on an example site? Or any one from the team?

**Comment by moonmeister** [comment]
I don't have time and we have no dedicated folks for working on this
plugin. Are you having trouble testing it?

**Pull request closed without merging**